governance-exercises/Le Bateleur - Interpretation Exercise 01/Assessment of Ambiguity.md

```diff
@@ -0,0 +1,20 @@
+# Interpretation Exercise 01
+
+---
+
+## Is there ambiguity in these clauses?
+
+I assess these clauses as ambiguous/overbroad. In making this assessment, I am guided by analogous principles of statutory interpretation whereby ambiguity/overbreadth is determined not only by referencing the language at issue, but also the specific context in which that language appears and the broader context of the statutory scheme as a whole.  
+
+The following language is problematic:
+
+- "...when this relates to unintended consequences or mistakes in the affected documents." (GOV 12.1)
+- "In case of unintended consequences or mistakes in the Atlas and Scope Artifacts causing Maker Governance to function incorrectly..." (GOV 12.1.1)
+
+The most glaring deficiency is the phrase "when this relates to". This is an [omnibus term](https://www.law.cornell.edu/wex/omnibus_clause#:~:text=A%20contractual%20provision%20that%20operates,a%20last%20will%20and%20testament.) - an indeterminate catch-all (i.e., "relating to", pertaining to", "concerning") common to legal artifacts. Its purpose is to sweep under the legal instrument's purview events and conditions not actually specified in the instrument. This phrase enables an expansive reading of GOV 12 that poses a significant risk to Maker Governance.
+
+Several governance provisions - such as the "bootstrapping" elements of which GOV 12 is one - are framed loosely to encompass contingencies the drafter cannot anticipate. While this may be advisable from a limited perspective, it is decidedly risky when granting a single class of actors (here, the Governance Facilitators) too wide a discretion to supersede customary governance protocols. 
+
+When drafting a bootstrapping provision such as GOV 12, one must provide a reasonable opportunity, i.e., fair notice, to discern what is prohibited and what is allowed. The clauses here are problematic because their application to a specific subject or purpose is not sufficiently restricted. A huge variety of agendas can theoretically be swept under the clauses' general and indefinite characterization. It would not be onerous for misaligned actors to posit *some relation* of their agenda to "unintended consequences or mistakes" in the rulebooks "causing Maker Governance to function incorrectly." Without more explicit and narrowly tailored standards of application, GOV 12 runs the risk of encouraging arbitrary, biased enforcement where Governance Facilitators adjudicate GOV 12 matters on an ad hoc basic. 
+
+Also, how is one to determine whether something counts as an "unintended" consequence/mistake? One would have to know the intent or preferred outcome behind each discrete unit of governance logic. But in the absence of meticulous, transparent and formal records detailing the drafters' precise intent (analogous to legislative committee reports), we become dangerously reliant on subjective, ad hoc adjudication on the part of centralized actors. 
```

Comment:
This is definitely the main issue I identify with the wording as is. Mainly the definitions of 'mistake' or 'unintended consequence'. Unintended consequence is definitely very broad. 

You're also correct that it requires knowledge of the intent of the writer (or judgement of the intent of the clauses based on surrounding clauses). Which sucks.